docs/_pages/releases.md

```diff
@@ -7,6 +7,13 @@ sidebar:
   nav: "sidebar"
 ---
 
+## 7.0.0
+
+### Breaking Changes
+
+* Dropped direct support for .NET Core 2.x and .NET Core 3.x - [#2302](https://github.com/fluentassertions/fluentassertions/pull/2302)
+* Dropped support for `NSpec3` test framework - [#2356](https://github.com/fluentassertions/fluentassertions/pull/2356)
```

Comment:
Found some more nspec:
* `Src/FluentAssertions/Execution/NSpecFramework.cs`
* `docs/_pages/about.md:56:* [NSpec](http://nspec.org/)`
* `docs/_pages/introduction.md:75:    <!-- Supported values: nunit, xunit2, mstestv2, nspec3 and mspec -->`
* `docs/index.html:57:Supports MSTest2, xUnit2, NUnit3, MSpec and NSpec3.`